contracts/StrategyManager.sol

```diff
@@ -179,40 +181,42 @@ contract StrategyManager is HubOwnable, UseTreasury, UseZap {
         uint8 dcaPercentage;
         uint8 vaultPercentage;
 
-        for (uint i = 0; i < _dcaInvestments.length; i++)
-            dcaPercentage += _dcaInvestments[i].percentage;
+        for (uint i = 0; i < _params.dcaInvestments.length; i++)
+            dcaPercentage += _params.dcaInvestments[i].percentage;
```

Comment:
Since investment count has a limit, could we use a `uint8` to declare the variable `i`?

Also (same for loop below):
```suggestion
        for (uint i = 0; i < _params.dcaInvestments.length; ++i)
            dcaPercentage += _params.dcaInvestments[i].percentage;
```